### DISCUSSION TITLE
Add Support for Scrobbling Tracks Shorter Than 30 Seconds

### DISCUSSION THREAD

**Discussion by herickdearaujo** [strawberrymusicplayer/strawberry#1590] [Ideas]
Currently, Strawberry Music Player follows the default Last.fm scrobbling behavior, which prevents tracks shorter than 30 seconds from being scrobbled. This limitation is an issue for users who listen to very short pieces, such as interludes, skits, or experimental music.

It would be great if Strawberry could offer more flexibility—perhaps an option to manually adjust the minimum scrobble length or a manual scrobble button for short tracks. This feature would align with the needs of users who want a complete listening history without omissions.

**Comment by jonaski** [comment]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
